Gemfile

```diff
@@ -166,7 +166,7 @@ group :development do
 
   # Simple command execution over SSH. Lightweight deployment tool.
   # Using our own version until https://github.com/mina-deploy/mina/pull/361 is merged.
-  gem 'mina', git: "git@github.com:samfundet/mina.git"
+  gem 'mina', git: "git://github.com/Samfundet/mina.git"
```

Comment:
Is it possible to tunnel the connection over https using e.g. https://github.com/Samfundet/mina.git? That way one might not have to compromise on security for convenience' sake.